packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -109,6 +137,46 @@ contract ControllerHelper is FlashControllerHelper, IERC721Receiver {
         emit FlashWMint(msg.sender, _vaultId, _wPowerPerpAmount, amountToFlashswap, _collateralAmount);
```

Comment:
I wonder if this should be passed in from f/e or just grabbed from the vault as the shortAmount